src/solvers/refinement/string_refinement.h

```diff
@@ -171,6 +171,9 @@ class string_refinementt: public bv_refinementt
   std::string string_of_array(const array_exprt &arr);
 };
 
+#define string_invariantt(reason) \
```

Comment:
I used lowercase because when the rest of the exception machinery lands, this will be a proper class and not a macro.